lib/new_relic/other_transaction.ex

```diff
@@ -7,7 +7,7 @@ defmodule NewRelic.OtherTransaction do
 
     NewRelic.add_attributes(
       pid: inspect(self()),
-      start_time: System.system_time(),
+      system_time: System.system_time(),
```

Comment:
The call `Reporter.start_transaction(:other)` automatically adds `system_time` now actually...

We can remove both `start_time` and `start_time_mono` from this call here

Reply:
The code as-is isn't a bug but it is setting 2 un-necessary attributes which we could remove